hack/jenkins/linux_integration_tests_virtualbox.sh

```diff
@@ -28,9 +28,10 @@ set -e
 OS_ARCH="linux-amd64"
 VM_DRIVER="virtualbox"
 JOB_NAME="VirtualBox_Linux"
+EXTRA_TEST_ARGS="-timeout-multiplier=2"
 EXPECTED_DEFAULT_DRIVER="kvm2"
 
 mkdir -p cron && gsutil -qm rsync "gs://minikube-builds/${MINIKUBE_LOCATION}/cron" cron || echo "FAILED TO GET CRON FILES"
 sudo install cron/cleanup_and_reboot_Linux.sh /etc/cron.hourly/cleanup_and_reboot || echo "FAILED TO INSTALL CLEANUP"
 
-source ./common.sh
+source ./common.sh 3h
```

Comment:
is this 3 hours really needed?

Reply:
this is mostly for testing, we're consistently getting really close to the 70 minute timeout even when we're succeeding

Reply:
lowered to 2 hours